### PR TITLE
Added Malaysia Localized data and fix Singapore able to generate StreetName from Array

### PR DESCRIPTION
**Pull request by shiroamada**
Added Malaysia Localized data

### CODE REVIEW

File: src/Faker/Provider/en_MY/Address.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    protected static $streetSuffix = array(
+        
+    );
+
+    // http://www.streetdirectory.com/malaysia/kl/asia_travel/street/
+    protected static $streetPrefix = array(
+        'Jalan', 'Lorong',
+    );
+
+    // http://www.streetdirectory.com/malaysia/kl/asia_travel/street/
```
Comment (fzaninotto) [inline src/Faker/Provider/en_MY/Address.php]: The content of this site is copyrighted. Please find a datasource compatible with the MIT license.

File: src/Faker/Provider/en_MY/Address.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        '{{streetPrefix}} {{streetName}}',
+    );
+
+
+    // https://en.wikipedia.org/wiki/Addresses_in_Malaysia
+    protected static $addressFormats = array(
+        "{{streetNumber}} {{streetAddress}},\n{{postcode}} {{townName}},\n{{country}}",
+    );
+
+    protected static $townName = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/en_MY/Address.php]: Not enough entropy, please find a larger list

File: src/Faker/Provider/en_MY/Address.php [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        'Johor', 'Kedah', 'Kelantan', 'Melaka', 'Negeri Sembilan', 'Pahang', 'Perak', 'Perlis',
+        'Penang', 'Sabah', 'Sarawak', 'Selangor', 'Terengganu', 'Kuala Lumpur', 'Labuan', 'Putrajaya',
+    );
+
+    protected static $postcode = array('#####');
+
+    protected static $country = array(
+        'Malaysia',
```
Comment (fzaninotto) [inline src/Faker/Provider/en_MY/Address.php]: No,  a random country should return any country in the world

File: src/Faker/Provider/en_MY/Address.php [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    );
+
+    public function streetPrefix()
```
Comment (fzaninotto) [inline src/Faker/Provider/en_MY/Address.php]: should be a static function. Same for many other functions in your code.

**Comment by pimjansen** [comment]
Closing this PR since it got stale. Please feel free to reopen if needed

**Pull request closed without merging**